components/molecules/HeaderComponent.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="container-fluid px-0 mb-5 pb-5">
-        <b-navbar class="navbar-padding-adjust" fixed="top" type="light" variant="light">
-            <b-container class="d-flex justify-content-between">
+        <b-navbar class="navbar-padding-adjust" fixed="top" type="light">
```

Comment:
shouldn't class .navbar-light work, why use type?

Reply:
i believe this is correct from lucas type = light will add navbar-light in behind the scenes. Just double check.

Reply:
removed the type. It's looking fine right now. Didn't add .navbar-light to the class too